src/main/java/com/thebois/models/beings/AbstractBeing.java

```diff
@@ -63,6 +77,10 @@ public AbstractBeing(
         this.finder = finder;
         setPath(pathFinder.path(startPosition, destination));
         Pawntastic.getEventBus().register(this);
+
+        currentHealth = MAX_HEALTH;
+        currentHunger = MAX_HUNGER;
```

Comment:
style

I think you can set these up in the field declarations, to make the constructor less cluttered.